FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -656,7 +664,7 @@ protected function setAttributes()
         $allAttributes[] = $xmlNewFlag;
 
         // Add free_shipping
-        if ($this->baseVariant->getShippingFree() == '') {
+        if ($this->baseVariant->getShippingFree() === '') {
```

Comment:
Checking the type strictly here may cause issues, because `getShippingFree` will return an `int`.